src/main/java/com/google/sps/scheduler/Scheduler.java

```diff
@@ -94,23 +111,32 @@
       }
     }
 
-    scheduleInterval(dayEndEpochMilliseconds, lastEnd, timeZone, tasksIterator, scheduledTasks);
-
-    return scheduledTasks;
+    scheduleInterval(dayEndEpochMilliseconds, lastEnd);
   }
 
   /**
    * Schedules the tasks in the free intervals between lastEnd and limit.
    */
-  private static long scheduleInterval(
-      long limit, long lastEnd, String timeZone, Iterator<Task> tasksIterator, List<Task> scheduledTasks) {
+  private long scheduleInterval(
+      long limit, long lastEnd) {
```

Comment:
Can be moved to line above.
Please auto-reformat this file.

Reply:
Interestingly, auto reformat did not do it for me.